src/BlockComponents/components/Form/Form.tsx

```diff
@@ -0,0 +1,18 @@
+import cn from "classnames";
+import * as React from "react";
+
+import "./Form.scss";
+
+interface FormProps {
+	labelPosition: "top" | "left";
+}
+
+export const Form: React.FC<FormProps> = ({ children, labelPosition }) => (
```

Comment:
Если сделать так, будет на 4 строчки меньше 🙃
```suggestion
export const Form: React.FC<FormProps> = ({ children, labelPosition = "left" }) => (
```

Reply:
это да:)) defaultProps как по мне более быстрее воспринимается

Reply:
Это, видимо, потому что ты так привык)